common/src/main/java/org/jboss/hal/testsuite/fragment/WindowFragment.java

```diff
@@ -70,18 +70,21 @@ public void waitUntilClosed() {
      * Closes wizard by clicking on cross in top right corner
      */
     public void close() {
-        By selector = By.className(PropUtils.get("modals.window.close.class"));
-        WebElement close = root.findElement(selector);
-
         int windowCount = Console.withBrowser(browser).getWindowCount();
 
-        close.click();
+        clickCross();
 
         waitUntilClosed(windowCount, true);
 
         closed = true;
     }
 
+    protected void clickCross() {
```

Comment:
I would suggest better name then `clickCross`, which would better indicate that the top the button at top right corner for closing is meant.

Thinking about possible names, I haven't figured out the best one yet, what about `clickWindowCloseButton` or something like that.